prover/zkevm/full.go

```diff
@@ -130,6 +130,7 @@ func fullZKEVMWithSuite(tl *config.TracesLimits, suite compilationSuite) *ZkEvm
 		Arithmetization: arithmetization.Settings{
 			Limits:            tl,
 			OptimisationLevel: &mir.DEFAULT_OPTIMISATION_LEVEL,
+			RelaxedMode:       false,
```

Comment:
Discussed this with Alex, and we think this mode needs to be added to the config (the prover .toml config file) so that we can turn it on off when necessary.

Reply:
Ok, I will need to figure out how to do that.  Stay tuned :)